Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/OnboardingGenerateKeyViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  OnboardingGenerateKeyViewController.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 26.05.21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class OnboardingGenerateKeyViewController: UITableViewController {
```

Comment:
We have onboarding screen for Import key which is almost identical to this except the cards, I think it's the time to create general onboarding view

Reply:
I thought about it too. I will try to play with it in the next pr